src/Data/Nat/DivMod.agda

```diff
@@ -223,6 +228,32 @@ m≥n⇒m/n>0 {m@(suc m-1)} {n@(suc n-1)} m≥n = begin
   n / d + (m * n) / d ≡⟨ cong (n / d +_) (*-/-assoc m d∣n) ⟩
   n / d + m * (n / d) ∎
 
+/-step : ∀ {m n n≢0} → m ≥ n → (m / n) {n≢0} ≡ 1 + ((m ∸ n) / n) {n≢0}
```

Comment:
This proof is missing from the CHANGELOG. I'd probably also rename it something a little bit more informative, e.g. `m/n≡1+[m∸n]/n`